x-pack/test/fleet_api_integration/apis/policy_secrets.ts

```diff
@@ -80,6 +199,38 @@ export default function (providerContext: FtrProviderContext) {
       return body.item;
     };
 
+    const enableSecrets = async () => {
+      try {
+        await kibanaServer.savedObjects.update({
+          type: GLOBAL_SETTINGS_SAVED_OBJECT_TYPE,
+          id: 'fleet-default-settings',
+          attributes: {
+            secret_storage_requirements_met: true,
+          },
+          overwrite: false,
+        });
+      } catch (e) {
+        throw e;
+      }
+    };
+
+    const disableSecrets = async () => {
+      try {
+        await kibanaServer.savedObjects.update({
+          type: GLOBAL_SETTINGS_SAVED_OBJECT_TYPE,
+          id: 'fleet-default-settings',
+          attributes: {
+            secret_storage_requirements_met: false,
+          },
+          overwrite: false,
+        });
+        // sleep to wait for refresh
+        await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
Weird we need that `wait_for` should be the default default refresh for saved object update

Reply:
Removed this line and seems to be working.